py/autoevals/oai.py

```diff
@@ -93,8 +103,9 @@ async def arun_cached_request(Completion=None, **kwargs):
 
     param_key = json.dumps(kwargs)
     conn = open_cache()
-    cursor = conn.cursor()
-    resp = cursor.execute("""SELECT response FROM "cache" WHERE params=?""", [param_key]).fetchone()
+    with ACACHE_LOCK:
```

Comment:
python docs say you need `async with` (https://docs.python.org/3/library/asyncio-sync.html#asyncio.Lock)?